src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/service/Et1VettingService.java

```diff
@@ -89,6 +92,21 @@ public void initialiseEt1Vetting(CaseDetails caseDetails) {
         caseDetails.getCaseData().setEt1VettingRespondentDetailsMarkUp(
             initialRespondentDetailsMarkUp(caseDetails.getCaseData()));
         populateRespondentAcasDetailsMarkUp(caseDetails.getCaseData());
+        initialEt1ReasonableAdjustments(caseDetails);
+    }
+
+    private static void initialEt1ReasonableAdjustments(CaseDetails caseDetails) {
+        CaseData caseData = caseDetails.getCaseData();
+        if (!ObjectUtils.isEmpty(caseData.getClaimantHearingPreference())
+            && !StringUtils.isEmpty(caseData.getClaimantHearingPreference().getReasonableAdjustments())) {
+            if (YES.equals(caseData.getClaimantHearingPreference().getReasonableAdjustments())) {
+                caseData.setEt1ReasonableAdjustmentsQuestion(YES);
+            } else {
+                caseData.setEt1ReasonableAdjustmentsQuestion(NO);
+            }
+            caseData.setEt1ReasonableAdjustmentsTextArea(
+                caseData.getClaimantHearingPreference().getReasonableAdjustmentsDetail());
+        }
```

Comment:
Could we use guard clauses here instead?

also `caseData.getClaimantHearingPreference()` and its `getReasonableAdjustments()` is called many times, might be better to extract as a variable.